_posts/2021-10-04-sharing-materials-from-ncwit-talk.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: Sharing resources on counteracting bias from recent NCWIT talk
+tags: [events, DEI, working groups]
+posted_by: David Nicholson
+---
+
+On Thursday, August 12th, US-RSE held the
+[second talk of our speaker series](https://us-rse.org/events/2021/2021-07-dei-speaker-series-ncwit)
+organized by the Diversity, Equity, and Inclusion (DEI) working group.
+We were joined by [Joanne Esch](https://us-rse.org/events/2021/2021-07-dei-speaker-series-ncwit)
+and [Andrea Bowen-Jones](https://ncwit.org/profile/andrea-bowens-jones/)
+from [The National Center for Women & Information Technology (NCWIT)](https://ncwit.org/).
+They spoke with us about interventions
+aimed at addressing the biases and barriers that
+face women and other underrepresented groups
+participating in the technical workforce.
+
+We would like to pass along some of the resources that the speakers shared.
+The first resource is the NCWIT's
+[critical listening guide](https://ncwit.org/resources/critical-listening-guide/).
+This guide addresses common misunderstandings that arise during discussions of
+the participation of women and other underrepresent group in technology.
```

Comment:
```suggestion
the participation of women and other underrepresented groups in technology.
```

Reply:
Thank you!

Reply:
I removed this phrase since I used it in the sentence above.